src/lib/schematics/update/test-cases/v5/attribute-selectors_expected_output.ts

```diff
@@ -0,0 +1,34 @@
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+
+const a = By.css('[cdkPortalOutlet]');
+const b = By.css('[cdkPortalOutlet]');
+
+// These should not change because we only update string literals that are
+// inside of a call expression.
+const c = 'cdkPortalHost';
+const d = 'portalHost';
+
+@Component({
+  template: `
+    <div cdkPortalOutlet="E"></div>
+    <div [cdkPortalOutlet]="myPortal"></div>
+  `,
+  styles: [
+    '[cdkPortalOutlet] {background: red}',
+    'div[cdkPortalOutlet] {color: blue}'
+  ]
+})
+class E {}
+
+@Component({
+  template: `
+    <div cdkPortalOutlet="F"></div>
+    <div [cdkPortalOutlet]="myPortal"></div>
+  `,
+  styles: [
+    '[cdkPortalOutlet] {background: red}',
```

Comment:
While creating the test-case I've started wondering if this actually makes sense. 

Even though the `cdkPortalOutlet` is usually placed on a `<ng-template>` and such CSS selectors do not make sense, we want to make sure that the **rule logic works**.

Reply:
I think it's fine